integrationTesting/alertListener.js

```diff
@@ -15,8 +15,9 @@ app.use(express.json()); // Used for smartThings wrapper
 app.post('/alert', async (req, res) => {
   var responderPhone = req.body.To
   var installationPhone = req.body.From;
+
   //Complete the Chatbot
-  axios.post('/sms', {
+  axios.post('/alert/sms', {
```

Comment:
This is the new route from the shared library. The Twilio number for each bathroom will need to be changed OR we need to add a redirect from `/sms` to `/alert/sms` when this is deployed.

Reply:
Made a script that can be used to update all the Twilio numbers. It's here in the Buttons PR: https://github.com/bravetechnologycoop/BraveButtons/pull/63/files#diff-416821bd63b8a30ec98017e06c469299a902869873e5dbe32ae43c470c23a942

Please let me know what you think. I've tested it on the Buttons Dev Twilio account.

Reply:
This isn't too onerous for ODetect - Primarily because the # bathrooms is much lower than the # of buttons. I'm happy to go with whatever you and Dave think is best for the buttons and adopt that for ODetect.

Reply:
The script is really cool :)

Reply:
Thanks!